modules/storage.js

```diff
@@ -0,0 +1,31 @@
+export default class Store {
+  static getBooks() {
+    let books = [];
+    if (localStorage.getItem('books') === null) {
+      return books;
+    }
+    books = JSON.parse(localStorage.getItem('books'));
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(ID) {
+    const books = Store.getBooks();
+    books.forEach((book, index) => {
+      if (book.id === Number(ID)) {
+        books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static generateId() {
+    return Math.floor(Math.random() * 100000000);
```

Comment:
- [ ] it would be better if you use the es6 `arrow function` instead of the `normal function` in order to practice to work with both